backend/app/views/spree/admin/style_guide/topics/typography/_colors.html.erb

```diff
@@ -0,0 +1,64 @@
+<%- snippet = capture do %>
+  <p>where the html example goes</p>
+<%- end %>
+
+<p>
+  The Solidus UI is made up mostly of a grey color palette with blue being used for interactive elements and red, green and yellow being used to give visual context to messaging.
+</p>
+
+<%-
+colors = [
```

Comment:
It would be really awesome to read the colors from the variables file, so they won't be needed to be updated in two places, once you update a value. 

I found a comment from the sass maintainer that could be helpful with this: https://github.com/sass/sass/issues/1246#issuecomment-42895688

Just a thought.